package.json

```diff
@@ -2,17 +2,21 @@
   "name": "react-dropzone",
   "description": "Simple HTML5 drag-drop zone with React.js",
   "main": "dist/index.js",
+  "module": "dist/es/index.js",
   "scripts": {
     "ci": "git-cz",
     "clean": "rimraf ./dist",
-    "build": "npm run clean && webpack",
+    "build": "npm run clean && webpack && npm run build:es",
+    "build:es": "BABEL_ENV=es babel ./src --out-dir ./dist/es --ignore spec.js",
     "start": "styleguidist server",
     "styleguide": "styleguidist build",
     "test": "npm run eslint:src && jest --coverage && npm run size",
+    "test:es": "JEST_TARGET=../dist/es/index jest --coverage",
+    "test:umd": "JEST_TARGET=../dist/index jest --coverage",
     "eslint:src": "eslint ./src ./examples ./*.js",
     "commitmsg": "commitlint -e",
     "precommit": "lint-staged",
-    "prepublish": "NODE_ENV=production npm run build && jest",
+    "prepublish": "NODE_ENV=production npm run build && npm run test:umd && npm run test:es",
```

Comment:
I'm also wondering if we should keep testing compiled targets. I know I'm the person who introduced that but with this PR it makes kind'a awful and I'm not sure it has enough value since Babel should be well-tested. Thoughts?

Reply:
I have recently had my share of broken npm dependencies.

One still had Flow types in transpiled output, other one was published as an application not a commonjs nor umd module. And one had React embedded in output, making it kind of big..

So I am pro target testing at the moment :)